cterasdk/convert/format.py

```diff
@@ -1,34 +1,59 @@
 import queue
 import json
+import copy
 from xml.etree.ElementTree import Element, SubElement, tostring
 from xml.dom import minidom
 
 from cterasdk.common import Item, Object
 from cterasdk.convert.xml_types import XMLTypes
 
 
-def tojsonstr(obj, pretty_print=True):
+_sdk_hidden = [
+    'password',
+    'awsSecretKey',
+    'sharedSecret',
+    'passPhraseSalt',
+    'encPassphrase',
+    'encryptedFolderKey',
+    'oldPassword',
+    'newPassword',
+    'secretkey',
+    'activationCode'
+]
+
+
+def _to_protected_dict(o):
+    ret = copy.deepcopy(o.__dict__)
+    for key in _sdk_hidden:
+        if key in ret:
+            ret[key] = '*** The Value is Hidden by the SDK ***'
+    return ret
+
+
+def tojsonstr(obj, pretty_print=True, no_log=False):
```

Comment:
I think no_log should be True by default, no?

Reply:
I'm ok with either. I thought hiding is less frequent than not hiding. We use `tojsonstr` in `show()` `show_multi()` and `show_query()` and I didn't want to make any changes to the implementation in other areas except the log prints

Reply:
I understand your point. But, I think it's cleaner to explicitly not hide details that we think should be hidden

Reply:
@ygalblum sounds good. i've incorporated the changes